src/node_sqlite.cc

```diff
@@ -515,6 +576,10 @@ class BackupJob : public ThreadPoolWork {
           return;
         }
       }
+      if (CheckAbortSignal()) {
+        HandleAbortError(resolver);
```

Comment:
Might be worth leaving a `TODO` comment here about `BackupJob` not implementing proper async context tracking (i.e. inherit from `AsyncWrap` and use `CallbackScope` to propagate async context), like the other `ThreadPoolWork` items do